test/__mocks__/text.mock.ts

```diff
@@ -152,8 +152,8 @@ export const UPDATE_OVERSEAS_ENTITY_PAGE_TITLE = "Confirm or update overseas ent
 export const CONFIRM_AND_CONTINUE_BUTTON_TEXT = "Confirm and continue";
 export const UPDATE_OVERSEAS_ENTITY_TITLE = "Update overseas entity";
 export const OE_NUMBER_FIELD_POPULATED = "value=\"OE123456\"";
-export const OVERSEAS_ENTITY_UPDATE_TITLE = "Update overseas entity";
-export const CHANGE_LINK_ENTITY_NAME = "/update-an-overseas-entity/entity#name";
+export const OVERSEAS_ENTITY_UPDATE_TITLE = "Check the overseas entity details";
+export const CHANGE_LINK_ENTITY_NAME = "/update-an-overseas-entity/entity#entity_name";
```

Comment:
again - could use new constant

Reply:
this is the test data though .. I like to be more explicit with it, rather than re-use config from source code. Because if the new variable someone accidentally changes the test wouldn't pick it up.

Reply:
ok